src/Search.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Products from './Products';
+
+class _Search extends React.Component{
+    constructor(){
+        super()
+        this.state =  {
+            searchItems: [],
+            inpu: ''
+        }
+    }
+     startSearch (e) {
+         const searchItems = this.props.products.filter(product => product.name.toLowerCase().includes(e.target.value.toLowerCase()))
+         
+         this.setState({ searchItems, input: e.target.value })
+    }
+
+    render(){
+       const { searchItems, input } = this.state
+
+   return( 
+       <div>
+           <form>
```

Comment:
do we need an onSubmit for the input form?

Reply:
no need because of no button there.